src/classes/CRLP_RollupCMT.cls

```diff
@@ -227,12 +227,11 @@ public class CRLP_RollupCMT {
             String valueLabel;
             List<String> rtLabels = new List<String>();
             if (fieldName == 'RecordTypeId') {
-                List<RecordType> rts = [SELECT Id, Name, DeveloperName FROM RecordType WHERE SobjectType = :objectName AND IsActive = true];
```

Comment:
I don't think so. All CRLP Unit Tests use record types in their testing and I believe Robot also does some rollup testing. Plus, this change just changes where the record type name comes from. I changed it here based on the TODO that I had put in about switching to the `getRecordTypeInfoByDeveloperName` method once it was available. 

The purpose of this method is to retrieve the label for the record type developer name to render in the CRLP UI (as opposed to during the actual rollups). What changes here is that the returned for the CLRP UI would now be translated instead of always in English.